packages/react/src/common/suspend.ts

```diff
@@ -0,0 +1,73 @@
+import { Client, ProviderEvents } from '@openfeature/web-sdk';
```

Comment:
No changes to these functions, I just extracted them so we could use them in multiple hooks.